packages/node/src/login/oidc/refresh/__mocks__/TokenRefresher.ts

```diff
@@ -50,6 +50,7 @@ export const mockDefaultTokenSet = (): TokenEndpointResponse => {
   return {
     accessToken: "some refreshed access token",
     idToken: JSON.stringify(mockIdTokenPayload()),
+    expiresAt: Date.now() + 3600,
```

Comment:
so 3600 is just the "refresh_extension_time" in miliseconds?

Reply:
Yup, that's just the token lifespan. On top of head, that's the value used by ESS (which isn't particularly important here ^^)